unit_tests/util/static_rtree.cpp

```diff
@@ -335,6 +335,34 @@ BOOST_AUTO_TEST_CASE(regression_test)
     BOOST_CHECK_EQUAL(result_ls.front().v, result_rtree.front().v);
 }
 
+// Bug: If you querry a point with a narrow radius, no result should be returned
```

Comment:
Why does this say `bug`? It looks like the code does quite that, not returning results?